ts/parts/SvgRenderer.ts

```diff
@@ -5504,6 +5528,10 @@ extend(SVGRenderer.prototype, /** @lends Highcharts.SVGRenderer.prototype */ {
                 );
             }
 
+            if (defined(options.longArc)) {
```

Comment:
@jon-a-nygaard @TorsteinHonsi 

I combined this logic with line 5498, just like Jon suggested, and later I noticed that the innerRadius has a constant clockwise (set to 0) which conflicts with the longArc functionality.

Reply:
@jon-a-nygaard @TorsteinHonsi,

I moved and combined this line logic with line 5498, just like Jon have suggested:

`longArc = options.end - start - Math.PI < proximity ? 0 : 1,` 
-> 
`    longArc = pick(options.longArc, options.end - start - Math.PI < proximity ? 0 : 1),
`
 And I found that functionality to render the inner arc has a constant clockwise set to 0, which navigated me to an interesting situation:  I needed to add this **options.longArc** for case when **dataLabel** is rendered in the lower half of the circle - to change the direction of the rendered path. I reproduced this case and see what happened with the innerRadius - it is impossible to control it. What is more, changing this constant 0 clockwise value to **pick(options.clockwise, 1)** - as it is for the outerRadius - breaks chart looks. 

[Demo](http://jsfiddle.net/BlackLabel/p1ch70t4/)

Reply:
> Should I leave it as it is or try to fix it?

Hmm, I would prefer to fix it if it is not too complicated. What do you think of this observation @TorsteinHonsi?

> the inner arc should have an opposite value for clockwise than the outer arc has.

Then I believe doing `1 - clockwise` on the innerArc would be the easiest solution. This will flip the value from 0 to 1, or the other way.